app/package.json

```diff
@@ -19,6 +19,7 @@
     "ts:check": "tsc --noEmit"
   },
   "dependencies": {
+    "@trussworks/react-uswds": "^4.0.0",
```

Comment:
Thanks for adding this! Can we call this out as a resource (and the suggested way) for utilizing the USWDS? The "Overview" section in `app/README.md` might be a good spot, but I don't feel strongly.

Reply:
added something!